Source/Tabs/Cancelling/Mixins/UndercutScan.lua

```diff
@@ -136,7 +136,9 @@ function AuctionatorUndercutScanMixin:SearchForUndercuts(auctionInfo)
     sortingOrder = {sortOrder = 4, reverseSort = false}
   end
 
-  C_AuctionHouse.SendSearchQuery(auctionInfo.itemKey, {sortingOrder}, true)
+  Auctionator.AH.Queue:Push(function()
```

Comment:
I think this should be a method in the AH namespace; otherwise we will sprinkle Queue calls throughout our system:
```lua
function Auctionator.AH.SendSearchQuery(itemKey, options, thisThing)
  Auctionator.AH.Queue:Push(function()
    C_AuctionHouse.SendSearchQuery(itemKey, options, thisThing)
  end)
end
```
which makes this code:
```lua
Auctionator.AH.SendSearchQuery(auctionInfo.itemKey, { sortingOrder }, true)
```